src/components/BlipGroupCard.vue

```diff
@@ -169,7 +175,9 @@ export default {
         photo: this.documents[0].photo,
         date: this.documents[0].date,
         hasNotification: this.showNotification(this.documents[0]),
-        status: this.documents[0].status
+        status: this.documents[0].status,
+        memberName: this.documents[0].document.metadata ? this.documents[0].document.metadata.memberName : '',
+        memberPhoneNumber: this.documents[0].document.metadata ? this.documents[0].document.metadata.memberPhoneNumber : ''
       }
       for (let i = 1; i < this.documents.length; i++) {
```

Comment:
Inside this loop you need to check if you can group the messages. You can only group messages from the same sender, let's say you have user 1 with phone number '1 555 111 111' and other user 2 with number '1 555 111 222', you can't group their messages together, you need to group sequential messages from user 1 and then another group for user 2.

Reply:
ok